app/assets/stylesheets/icon_customizations.scss

```diff
@@ -25,7 +25,7 @@
   content: "\e602";
 }
 .pficon-close:before {
-  color: $color-critical;
+  // color: $color-critical;
```

Comment:
If this line is no longer needed, can we just delete it instead of commenting it out?